designs/tripleo/osc-tripleo.md

```diff
@@ -176,6 +183,30 @@ path/directory on overcloud node and the TLS certificate is copied to this locat
 
 >>c. Do we need to hard code the target path for storing the certificate? If so how is this used by puppet?
 
+#### OSC Network Related Configuration
```

Comment:
OSC Network Related Configuration -> OSC Network Configuration for consistency with the other similar headers.

Reply:
Done.